ui/imports/shared/controls/InformationTag.qml

```diff
@@ -4,32 +4,44 @@ import QtQuick.Controls 2.14
 
 import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
 
 import utils 1.0
 
 Control {
+    id: root
+
     property alias image : image
     property alias iconAsset : iconAsset
     property alias tagPrimaryLabel: tagPrimaryLabel
     property alias tagSecondaryLabel: tagSecondaryLabel
     property alias controlBackground: controlBackground
     property alias rightComponent: rightComponent.sourceComponent
+    property bool loading: false
 
-    implicitWidth: 66
-    implicitHeight: 26
-    horizontalPadding: Style.current.halfPadding
-    verticalPadding: 5
-
-    background: Rectangle {
+    property var customBackground: Rectangle {
         id: controlBackground
         color: "transparent"
         border.width: 1
         border.color: Theme.palette.baseColor2
         radius: 36
     }
 
+    QtObject {
+        id: d
+        property var loadingComponent: LoadingComponent {}
+    }
+
+    implicitWidth: 66
+    implicitHeight: 26
+    horizontalPadding: Style.current.halfPadding
+    verticalPadding: 5
+
+    background: root.loading ? d.loadingComponent : root.customBackground
```

Comment:
Why not make the `background` a simple `Loader` and choose its `sourceComponent` based on `root.loading`?

Reply:
Can do too